adapter/src/test/kotlin/org/javacs/ktda/SampleMavenWorkspaceTest.kt

```diff
@@ -0,0 +1,13 @@
+package org.javacs.ktda
+
+/**
+ * Tests the basic debugging scenarios
+ * using a sample maven application.
+ * In addition to any application specific scenarios
+ */
+class SampleMavenWorkspaceTest : GlobalWorkspaceTest(
```

Comment:
It allows for Maven/Gradle/other workspace specific testcases in the future.
While all tests that we want for every workspace can stay in the parent.